compiler/src/org.graalvm.compiler.serviceprovider/src/org/graalvm/compiler/serviceprovider/JavaVersionUtil.java

```diff
@@ -35,6 +35,7 @@ public final class JavaVersionUtil {
      * The value of calling {@link Version#feature()} on {@link Runtime#version()}.
      */
     public static final int JAVA_SPEC = Runtime.version().feature();
+    public static final String JAVA_VERSION = Runtime.version().toString();
```

Comment:
There is no need to introduce this field. The `JavaVersionUItil` class is only for intended for numeric version checks.